Adding extra link (#47)
comp129/comp55: docs/labs/Supplemental/lucidchart.md

# Lucid Charts

Before getting familiar with this, keep in mind that the main focus of the course will be **PLANTUML**.

> This is just another extra information that might be helpful for your future projects and classes.

Reviewer: ```suggestion
This is just some additional information that might be helpful for your future projects and classes.
```

Here I would like to introduce the [LucidChart](https://www.lucidchart.com) online software.
Here are some functionalities of it:

- *Easily style shapes*
![select-all-shapes](https://user-images.githubusercontent.com/101157079/223881056-673184ab-fc74-4f56-b057-230ca2148f0c.gif)
- *Organizing diagrams with containers*
![diagrams with containers](https://user-images.githubusercontent.com/101157079/223914040-62d6f66b-f60d-4e08-8384-a8ae9c11829e.png)

> - *Ceate clean diagrams*

Reviewer: ```suggestion
- *Create clean diagrams*
```

![smart-lines](https://user-images.githubusercontent.com/101157079/223919857-a878c88d-2e39-4203-983d-f2e0999a5995.gif)
- *And most importantly, collaborate with teammates*
Please follow the following animation for setting it up. Good luck!

After the log in this window will show up on the screen

![new button navigation](https://user-images.githubusercontent.com/101157079/223916912-1b5d66c5-aeb2-4dcf-b1f7-632cae20697c.png)

> The shortest way of creating a new project is using **New** button. There you will have 2 options

Reviewer: ```suggestion
The shortest way of creating a new project is using the **New** button. There you will have 2 options
```

![creating a new project](https://user-images.githubusercontent.com/101157079/223920897-e064e753-1d37-4f38-b751-0971fef911a3.png)

> I would suggest to use blank document if you are planning to create UML diagrams.

Reviewer: ```suggestion
I would suggest to use a blank document if you are planning to create UML diagrams.
```

> On the left side, second button will take you there directly

Reviewer: ```suggestion
On the left side, the second button will take you there directly
```

![blank canvas for uml diagrams](https://user-images.githubusercontent.com/101157079/223921908-3c754b43-11ca-4193-9d96-e9e9951520d3.png)

> However, for different projects and chart types you can go to the templates option will open

Reviewer: ```suggestion
However, for different projects and chart types you can go to the templates option which will open
```

> many different kinds of graphs that might be useful for any projects

Reviewer: ```suggestion
many different kinds of graphs that might be useful for any project.
```
